pyproject.toml

```diff
@@ -22,41 +22,80 @@ dynamic = ["version"]
 dependencies = [
     "xarray>=2025.1.1",
     "numpy>=2.0.0",
-    "packaging",
     "universal-pathlib",
     "numcodecs",
     "ujson",
+    "packaging",
 ]
 
 [project.optional-dependencies]
-hdf_reader = [
+remote = [
     "fsspec",
+    "requests",
+    "aiohttp",
+    "s3fs",
+]
+# non-kerchunk-based readers
+hdf = [
+    "virtualizarr[remote]",
     "h5py",
     "hdf5plugin",
     "imagecodecs",
     "imagecodecs-numcodecs==2024.6.1",
-    "numcodecs"
+]
+# kerchunk-based readers
+hdf5 = [
+    "virtualizarr[remote]",
+    "kerchunk>=0.2.8",
+    "h5py",
 ]
 netcdf3 = [
+    "virtualizarr[remote]",
     "kerchunk>=0.2.8",
     "scipy",
 ]
 fits = [
+    "virtualizarr[remote]",
     "kerchunk>=0.2.8",
     "astropy",
 ]
+# un-implemented readers
+# tiff = [
+#     "virtualizarr[remote]",
+#     "kerchunk>=0.2.8",
+#     "tifffile",
+# ]
+# grib = [
+#     "virtualizarr[remote]",
+#     "kerchunk>=0.2.8",
+# ]
```

Comment:
These commented-out lines are going to appear in the docs - https://virtualizarr--309.org.readthedocs.build/en/309/installation.html#optional-dependencies.

Should I (a) remove them or (b) add issue links to encourage people to contribute those readers?

Reply:
I think we should remove them because I don't think it's a certainty that those readers will be built based on kerchunk and its dependencies. e.g., I have it on my roadmap for the next 2 months to implement a reader based on https://github.com/developmentseed/aiocogeo-rs/.